client/package.json

```diff
@@ -12,6 +12,9 @@
     }
   ],
   "devDependencies": {
+    "babel-core": "^6.10.4",
+    "babel-loader": "^6.2.4",
+    "babel-preset-es2015": "^6.9.0",
     "btoa": "^1.1.2",
     "grunt": "^0.4.2",
```

Comment:
To replace grunt with gulp you mean?

I personally prefer gulp, but since ```superdesk-client-core``` is relying on grunt, I think we should keep on using it.